changelog.md

```diff
@@ -18,6 +19,7 @@
 - Add: [Script] ScriptEventCompanyRename (#12878)
 - Add: Ukrainian Hryvnia currency (#12877)
 - Add: Convert 32bpp-only sprites to 8bpp when needed (#11602)
+- Change: [Script] Start GS (but don't run it) when generating world in scenario editor (#13961)
```

Comment:
Isn't this a "Fix"?

Reply:
Good question. The PR had it as Change, probably because it doesn't mention any real problem that is fixed, just an inconsistency. I'll wouldn't mind moving it to Fix though.